DEVELOPMENT.md

```diff
@@ -26,6 +26,13 @@
     For development purposes it can be easier to run with SQLite, to avoid extra dependencies.
     To run RMT with SQLite, switch the database adapter in `config/rmt.yml` to `sqlite3`.
 
+4. **Background jobs:** If you plan on implementing background jobs (via [Active Job](https://guides.rubyonrails.org/v6.1/active_job_basics.html)), you will need to install or utilize a service compatible with the Redis API. We recommend [Valkey](https://valkey.io/), a BSD-licensed server, which is packages for (open)SUSE distributions:
+
```

Comment:
s/packages/packaged/

Reply:
Resolved in [b10032a](https://github.com/SUSE/rmt/pull/1315/commits/b10032ad0b0e1c3be9006d8583eaeaf37dd44344).